pkg/volume/volume.go

```diff
@@ -282,6 +289,9 @@ func (v *Operation) Read() (*v1alpha1.CASVolume, error) {
 
 		// extract the sc name
 		scName = strings.TrimSpace(pv.Spec.StorageClassName)
+
+		// extract the storage engine
+		storageEngine = pv.Labels[string(v1alpha1.CASTypeKey)]
```

Comment:
we can initialised and assign `stroageEngine` here itself ?
```suggestion
		storageEngine := pv.Labels[string(v1alpha1.CASTypeKey)]
```